resources/assets/js/app/routes.js

```diff
@@ -26,23 +28,14 @@ export default (
         <Route path="trip/create" component={ CreateTrip } />
         <Route path="trip/edit/:id" component={ Vehicles /*TripEdit*/ } />
 
-        <Route path="registration" component={ RegisterForm } />
-        <Route path="registration/success" component={ RegisterSuccess } />
-        <Route path="verification" component={ RegisterVerify } />
+        <Route path="registration" component={ RegisterForm } onEnter={ alreadyAuth } />
```

Comment:
rename alreadyAuth method to more appropriate, name should reflects logic that this method performs, in this case this method checks access and provides business logic if user doesn't authorized